### DISCUSSION TITLE
FR: Config to change the border style

### DISCUSSION THREAD

**Discussion by chrisgrieser** [bennypowers/nvim-regexplainer#27] [Q&A] [answered]
regexplainer seems to use nui popups that always have the "shadow" border style. Since all my other popups have a different border style, it would be nice if they were consistent, meaning an option to change the popup border style would be very useful.

**Comment by bennypowers** [comment, marked as answer]
https://github.com/bennypowers/nvim-regexplainer#popup-below-cursor


> You can customize the popup window by specifying `options.popup.border`,
> which is a table of [popup options from nui](https://github.com/MunifTanjim/nui.nvim/tree/main/lua/nui/popup#border).
> Any options specified for `options.popup` will also override the defaults.
> 
> ```lua
> require'regexplainer'.show {
>   display = 'popup',
>   popup = {
>     border = {
>       padding = { 1, 2 },
>       style = 'solid',
>     },
>   },
> }
> ```
> 
> You could use this to, for example, set a different border based on the state of your editor.

You could also pass `popup` in the options to `setup { }`

**Comment by chrisgrieser** [comment]
ah, got it. Wasn't aware you could just put `popup` into the `setup` function, as is wasn't listed in the config options. Thanks!
